op-mode-definitions/poweroff.xml

```diff
@@ -2,37 +2,45 @@
 <interfaceDefinition>
   <node name="poweroff">
     <properties>
-      <help>Poweroff the system</help>
+      <help>Shutdown the system</help>
```

Comment:
In the UNIX world there is a difference between shutdown and poweroff. Shutdown will only shutdown the Kernel but the system will continue to run. With poweroff we really shutdown the Kernel and the power-off the system. Please keep the old help text.

Reply:
undoing change :)